.github/CODEOWNERS

```diff
@@ -1,2 +1,2 @@
-* @bigcommerce/frontend @bigcommerce/product-design
-packages/docs @bigcommerce/dev-docs-team @bigcommerce/frontend @bigcommerce/product-design
```

Comment:
I'm not sure if this line should actually change. It might be fine for the entire frontend group to be able to approve documentation changes?

Reply:
@PascalZajac It looks like we don’t actively contribute to the docs package, as updates mostly happen alongside changes to the big-design package. Given that, these changes seem valid.

Since the documentation is public and directly tied to BigDesign components, it makes sense for it to have the same focus and ownership as the big-design package.